controllers/help_request.js

```diff
@@ -0,0 +1,48 @@
+var handle = require("../utils/error_handling");
+var ObjectId = require("mongodb").ObjectId;
+var HelpRequestModel = require("../models/help_request").model;
+var UserModel = require("../models/user").model;
+var NotificationService = require("../services/notification.service");
+var UserService = require("../services/user.service");
+
+const addHelpRequest = async (req, res) => {
+  let userId = req.body.userId;
+  let user = null;
+
+  try {
+    user = await UserService.findUserById(userId);
+  } catch (err) {
+    handle.badRequest(res, err.message);
+  }
+
+  const responders = user.responders;
+
+  let help_request = new HelpRequestModel({
+    userId: userId,
+    responderId: null,
+    status: "open",
+    responders: responders
+  });
+
+  try {
+    await help_request.save();
+  } catch (err) {
+    handle.internalServerError(res, "Cannot create help request.");
+  }
+
+  await NotificationService.sendBatchNotifications(user, help_request);
```

Comment:
This is all still a "TODO" right

Reply:
If you are asking about the sendBatchNotifications, yes. It still needs to handle potential errors coming back as a Promise in the ticket chunks. We need that data for analytics as well later (# notifications that were sent successfully / # responders contacted)